config/tasks/multi-task-config.yaml

```diff
@@ -0,0 +1,17 @@
+# Schema version.
+version: "1.0"
+
+# Task title for display purposes.
+title: "Multi task example"
+
+# Layout of the steps that define this task.
+tasks:
+  # Item format:
+  # - id: Identifying integer for the task.
+  # - config_file: The task config file representing this task.
+  - id: 0
+    config_file: "./config/tasks/task_steps_config-recipe_coffee_trimmed_v3_allows_skip.yaml"
```

Comment:
SInce these are relative paths, it should be noted what this is expected to be relative to, e.g. where the tool is run, `${ANGEL_WORKSPACE}`, etc.

Reply:
Note added